spyderlib/utils/introspection/rope_plugin.py

```diff
@@ -167,7 +167,7 @@ def handle_info(self, cts, doc_text, source_code, offset):
                     calltip = getsignaturefromtext(doc_text, obj_name)
         if not obj_fullname:
             obj_fullname = sourcecode.get_primary_at(source_code, offset)
-        if obj_fullname and not obj_fullname.startswith('self.') and doc_text:

```

Comment:
This allows us to provide the other information while still showing the "No documentation" banner.